eth/eventdatahandler/task.go

```diff
@@ -0,0 +1,101 @@
+package eventdatahandler
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+
+	ssvtypes "github.com/bloxapp/ssv/protocol/v2/types"
+)
+
+type Task interface {
+	Execute() error
+}
+
+type AddValidatorTask struct {
+	executor  TaskExecutor
```

Comment:
this could probably be a thinner interface containing only `AddValidator` or even just a function. 
this could applies to all tasks below, it might be useful for tests.

Reply:
Actually functions need type aliases to look good which is not shorter than interfaces, so let's proceed with interfaces. Fixed